frontend/libs/studio-components/src/components/StudioRecommendedNextAction/StudioRecommendedNextAction.tsx

```diff
@@ -0,0 +1,46 @@
+import { StudioButton } from '../StudioButton';
+import React from 'react';
+import classes from './StudioRecommendedNextAction.module.css';
+import { StudioCard } from '../StudioCard';
+
+export interface StudioRecommendedNextActionProps {
+  onSave: React.MouseEventHandler<HTMLButtonElement>;
+  saveButtonText: string;
+  onSkip: React.MouseEventHandler<HTMLButtonElement>;
+  skipButtonText: string;
+  title: string;
+  description: string;
+  hideSaveButton?: boolean;
+  children?: React.ReactNode;
+}
+
+export const StudioRecommendedNextAction = ({
+  onSave,
+  saveButtonText,
+  onSkip,
+  skipButtonText,
+  title,
+  description,
+  hideSaveButton = false,
+  children,
+}: StudioRecommendedNextActionProps): React.ReactElement => {
+  return (
+    <StudioCard>
+      <StudioCard.Header>{title}</StudioCard.Header>
+      <StudioCard.Content>
+        <p className={classes.description}>{description}</p>
```

Comment:
We use <StudioParagraph/>, a wrapper around @digidr/designsystemet paragraph, instead of native tags. This ensures we get the correct styling, sizing, and spacing. :)